plugins/mac-capture/mac-sck-video-capture.m

```diff
@@ -606,11 +319,11 @@ typedef void (^shareable_content_callback)(SCShareableContent *,
 
 fail:
 	obs_leave_graphics();
```

Comment:
Removed. Technically should be part of the first refactor commit since this is in video-capture but resolving for now. If I have to do commit surgery later, or if you want me to, I will add it to the refactor commit.